README-template.md

```diff
@@ -790,6 +790,19 @@ The API Supports a number of options to configure the read
      </td>
      <td>Read/Write</td>
    </tr>
+  <tr>
+     <td><code>kmsKeyName</code>
```

Comment:
maybe a more descriptive name specifying its purpose (I don't know if there will be other kms keys required for the connector in the future?

Reply:
Renamed kmsKeyName to destinationTableKmsKeyName